index.html

```diff
@@ -400,6 +400,28 @@ <h3>
       <dfn class="export" data-dfn-for="DigitalCredential">[[\discovery]]</dfn>
       whose value is "remote".
     </p>
+    <h3>
+      `isProtocolSupportedByClient()` Method
+    </h3>
+    <p>
+      Digital credential verifiers and issuers use this static method to
+      determine which [=digital credential/exchange protocols=] the client
+      understands.
```

Comment:
I think what's important here is not whether the client "understands" the protocol but whether it will permit it to be passed on to the platform. Chromium current plans to pass on anything without necessarily understanding most things. What if we said "the client supports sending to credential providers"?

Reply:
Yeah, I actually had that originally and can't remember why I changed to understands. I'll switch it back.